lib/active_model/serializer.rb

```diff
@@ -177,6 +177,10 @@ def json_key
       end
     end
 
+    def as_json
+      self.class.adapter.new(self).as_json
```

Comment:
@bf4 I partially agree with u on this.
The goal here is to make it simpler to use AMS on tests and outside controllers in general.
Indeed the implementation at [your comment](https://github.com/rails-api/active_model_serializers/issues/878#issuecomment-96073030) is clean but not as easy as ```as_json```.
In the other hand, the PR that you mentioned might provide another solution, so I'm leaving this open some more days for now.